lib/pundit/policy_finder.rb

```diff
@@ -44,6 +44,8 @@ def find
           object
         elsif object.is_a?(Symbol)
           object.to_s.classify
+        elsif object.is_a?(Array)
+          object.map(&:to_s).join('/').to_s.classify
```

Comment:
Can't this be put more simply as: `object.join('/')` since `to_s` is implicitly called or am I missing something?

```
2.2.0 :001 > [:a, :b].join('/')
 => "a/b" 
2.2.0 :002 > class Foo
2.2.0 :003?>   end
 => nil 
2.2.0 :004 > [Foo.new, :b].join('/')
 => "#<Foo:0x007f861a1686e8>/b" 
```